source/features/extend-diff-expander.css

```diff
@@ -19,3 +19,29 @@
 	);
 	cursor: pointer;
 }
+
+/* Support react commit view */
+.rgh-extend-diff-expander .diff-line-row:has(button[data-direction]):hover {
+	color: var(
+		--diffBlob-hunkNum-fgColor-hover,
+		var(--fgColor-onEmphasis, fuchsia)
```

Comment:
By the way it might be a good time to also use `fuchsia` in the previous version